services/src/test/java/org/jfrog/artifactory/client/ItemTests.java

```diff
@@ -239,4 +255,24 @@ private void deleteAllRelatedRepos() throws IOException {
         deleteRepoIfExists(NEW_LOCAL_FROM);
         deleteRepoIfExists(NEW_LOCAL_TO);
     }
+
+    private void calcSha256ForItem(String repoName, String path) {
+        ObjectMapper mapper = new ObjectMapper();
+        Map<String, Object> map = null;
+        try {
```

Comment:
It might be better not to swallow this exception, but in case it is thrown, let it be thrown by this method.